README.md

```diff
@@ -1 +1,116 @@
-.
+![ETHERNAUT BANNER LOGO](img/banner.jfif)
+
+<h1 align="center">Educational Resources</h1>
+
+*<p align="center">A curated list of educational resouces to get you started on your <br />journey to become an Ethernaut </p>*
+
+<br /><h1 align="center">Categories</h1>
+
+ 
+
+
+<details>
+  <summary><span style="font-size: 1.5em;">🎮 Games</span></summary>
+
+<br />
+
+| **Description** | **Link** |
+| ----------- | ----------- |
+| Ethernauts | https://ethernaut.openzeppelin.com/ |
+| CryptoZombies | https://cryptozombies.io/ |
+| Blockchain security games & challenges | https://github.com/openblocksec/blocksec-ctfs |
+| Damn Vulnerable DeFi | https://www.damnvulnerabledefi.xyz/ |
+| Capture the Ether | https://capturetheether.com/ |
+
+</details>
+
+<details>
+  <summary><span style="font-size: 1.5em;">🎓 Tutorials</span></summary>
+
+<br />
+
+| **Description** | **Link** |
+| ----------- | ----------- |
+| Hardhat | https://hardhat.org/tutorial/ |
+| Let's Do Defi Guides | https://letsdodefi.com/ |
+
+</details>
+
+<details>
+  <summary><span style="font-size: 1.5em;">📑 Articles</span></summary>
+
+<br />
+
+| **Description** | **Link** |
+| ----------- | ----------- |
+| The Complete Guide to Full Stack Ethereum Development      | https://dev.to/dabit3/the-complete-guide-to-full-stack-ethereum-development-3j13 |
+| How to become a DeFi Developer - Kerman Kohli| https://defiweekly.substack.com/p/how-to-become-a-defi-developer |
+| A Guide for Developers Interested in Learning Blockchain Development| https://www.linumlabs.com/articles/a-guide-for-developers-interested-in-learning-blockchain-development |
+| Taking undercollateralized loans for fun and for profit | https://samczsun.com/taking-undercollateralized-loans-for-fun-and-for-profit/ |
+| Resources for learning smart contract security | https://immunefi.com/learn/ |
+| Most common smart contract bugs of 2020 | https://medium.com/solidified/most-common-smart-contract-bugs-of-2020-c1edfe9340ac |
```

Comment:
Could you add a few of mine here 🙏 

This is a bit hardcore, but very useful
https://blog.openzeppelin.com/deconstructing-a-solidity-contract-part-i-introduction-832efd2d7737/

This one's nice for The Graph
https://theethernaut.substack.com/p/developer-superpowers-with-thegraph